Example/Example/LoadingViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LoadingViewController.swift
+//
+//  Copyright (c) 2019 Evolv Technology Solutions
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+class LoadingViewController: UIViewController {
+
+    @IBOutlet weak var label: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        EvolvClientHelper.shared.didChangeClientStatus = { [weak self] status in
+            switch status {
+            case .ready:
+                self?.performSegue(withIdentifier: "showMain", sender: nil)
+            case .failed:
+                self?.label?.text = "Evolv Client - Failed to load data"
```

Comment:
client should always be available, even if allocations fail. this is why we fall back to use default in subscribe if allocations fail.